pyomo/core/base/suffix.py

```diff
@@ -105,7 +105,7 @@ class SuffixDataType(enum.IntEnum):
     FLOAT = 4
 
 
-class SuffixDirection(enum.IntEnum):
+class SuffixDirection(IntEnum):
```

Comment:
You didn't move the descriptions of the enum options out of the docstring like you did for others. Is this intentional?

Reply:
I was mostly focused on silencing warnings from Sphinx (I would like to turn on the "map warnings to errors" in the Sphinx configuration to protect us from our future selves).  These weren't tossing warnings because they are not actually documenting the Enum members.  In the long run, we should definitely move the documentation so that it will be picked up -- but in the spirit of managing the complexity / size of PRs I was going to punt that to later.